index.js

```diff
@@ -58,7 +57,7 @@ module.exports = function (request, options) {
   return ch
 
   function onField(name, val) {
-    var args = slice.call(arguments)
+    var args = [name, val]
```

Comment:
Remove slice, use `onField(fieldname, val, fieldnameTruncated, valTruncated)` will be better.

```js
function onField(name, val) {
  var args = [fieldname, val, fieldnameTruncated, valTruncated]
}
```